packages/contracts/contracts/DependencyRegistryV0.sol

```diff
@@ -42,628 +44,643 @@ contract DependencyRegistryV0 is
 
     uint8 constant AT_CHARACTER_CODE = uint8(bytes1("@")); // 0x40
```

Comment:
@yoshiwarab since this is a constant, I'm not sure if this ends up being an issue without using gap storage of diamond storage patterns in `DependencyRegistryV0Storage`.
That being said, I think the storage abstraction is a good thing overall here, but I think we would benefit from using a diamond storage pattern in a follow-on PR. So overall, this is okay to merge as-is as long as we do that follow-up work.

Reply:
resolving due to #1224 follow-on 🙌